packages/react-dom-bindings/src/server/ReactFizzConfigDOMLegacy.js

```diff
@@ -46,16 +51,20 @@ export type RenderState = {
   importMapChunks: Array<Chunk | PrecomputedChunk>,
   preloadChunks: Array<Chunk | PrecomputedChunk>,
   hoistableChunks: Array<Chunk | PrecomputedChunk>,
-  preconnects: Set<any>,
-  fontPreloads: Set<any>,
-  highImagePreloads: Set<any>,
-  // usedImagePreloads: Set<any>,
-  precedences: Map<string, Map<any, any>>,
-  stylePrecedences: Map<string, any>,
-  bootstrapScripts: Set<any>,
-  scripts: Set<any>,
-  bulkPreloads: Set<any>,
-  preloadsMap: Map<string, any>,
+  preconnects: Set<Resource>,
+  fontPreloads: Set<Resource>,
+  highImagePreloads: Set<Resource>,
+  // usedImagePreloads: Set<Resource>,
+  precedences: Map<string, PrecedenceQueue>,
```

Comment:
Nit: I don't really like the name precedence here because 1) we can easily have other fizz concepts named that and this is specifically about styles 2) the ultimate terminal content of this isn't the flushing precedence it's the stylesheets. It happens to be *grouped* by precedence but it's not about flushing the styles.

Proposal:

`styles: Map<string, StyleQueue>`

It's a queue for styles, it contains a queue of styles which are grouped by precedence. The queue happens to have a chunk attribute annotating the precedence of this queue but its primary purpose is to be a queue of styles.

Reply:
yeah I like this